.rustfmt.toml

```diff
@@ -0,0 +1,13 @@
+use_small_heuristics = "Default"
+force_explicit_abi = true
```

Comment:
`true` is the default.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#force_explicit_abi